modules/core/src/core/animation-loop.js

```diff
@@ -190,6 +197,13 @@ export default class AnimationLoop {
     return this;
   }
 
+  getCanvasDataURL() {
```

Comment:
Nit: This is technically a very correct name, but is it the most helpful name we can share with the user? 

Maybe worth ideating around something like `captureScreen()` or `captureCanvas`? 

Or if in keeping with the underlying API maybe just `toDataURL()`?

Just grist for the mill...

Reply:
I was thinking about the first two, but felt it was less clear about what the method would return (an image? a canvas? pixels?). `toDataURL` sounds good to me.

Reply:
Fine. 

It could still return a data url, it would be described in the docs. In this version, the name would focus on what the method does, rather than how the result happens to be encoded.

That said, `toDataURL()` does fits with the luma.gl philosophy of providing thin, transparent class wrappers on top of the underlying objects.